src/v/rpc/simple_protocol.cc

```diff
@@ -113,8 +113,12 @@ simple_protocol::dispatch_method_once(header h, net::server::resources rs) {
                   rs.probe().method_not_found();
                   netbuf reply_buf;
                   reply_buf.set_status(rpc::status::method_not_found);
-                  return send_reply(ctx, std::move(reply_buf))
-                    .then([ctx]() mutable { ctx->signal_body_parse(); });
+                  return ctx->res.conn->input()
```

Comment:
Is there anything server-side that indicates to users that such an error was hit? If not, consider adding a debug log message that includes the method ID, and maybe some info about the transport if we have it

Maybe something in client_probe too, though not sure it's worth adding given we expect this to be pretty uncommon

Reply:
huh, I really thought we logged logged all error replies, but indeed I don't see this. I think this is worth doing, especially for these couple of cases that should be very uncommon.